.cirrus.yml

```diff
@@ -72,16 +72,17 @@ check_size_task:
     GPG_SIGNING_KEY: VAULT[development/kv/data/sign data.key]
     GPG_SIGNING_PASSPHRASE: VAULT[development/kv/data/sign data.passphrase]
     folder: node_modules
-    SIZE_LIMIT: 30720
```

Comment:
What is the hard limit by azure devops again? I think 50 MB right? And for those on Azure DevOps Server they have a workaround for large file sizes I believe

Reply:
Exactly `50M` for AzDO Services and as low as `20M` for some TFS servers (but there are work arounds)